cmd/kaniko-docker/main.go

```diff
@@ -119,6 +119,11 @@ func main() {
 			Usage:  "Cache timeout in hours. Defaults to two weeks.",
 			EnvVar: "PLUGIN_CACHE_TTL",
 		},
+		cli.StringFlag{
```

Comment:
use a boolean flag instead of string flag

Reply:
oh my bad! done!